mappings/base_mapping.py

```diff
@@ -29,6 +29,22 @@ def applyMapping(self):
 
     def applyFormatting(self):
         return self.record
+    
+    def format_identifiers(self, identifiers, source):
```

Comment:
@jackiequach @mitri-slory what are our thoughts on moving away from classes here and being purely functional? This could be a function inside a identifier utils file.

Reply:
I can agree with our talk from earlier that we can change the current mapping we have to a more functional one.